components/tools/OmeroWeb/omeroweb/webclient/webclient_gateway.py

```diff
@@ -1382,8 +1383,7 @@ def updateGroup(self, group, name, permissions, owners=list(),
         # Should we update updateGroup so this would be atomic?
         admin_serv.updateGroup(up_gr)
         if permissions is not None:
-            logger.warning("WARNING: changePermissions was called!!!")
```

Comment:
Just as a reminder: this WARNING was here because the form was unintentionally calling `updatePermissions` on every save.